sys/net/network_layer/ng_sixlowpan/ctx/ng_sixlowpan_ctx.c

```diff
@@ -36,7 +36,7 @@ static char ipv6str[NG_IPV6_ADDR_MAX_STR_LEN];
 static inline bool _still_valid(unsigned int id)
 {
     _update_lifetime(id);
-    return (_ctxs[id].ltime > 0);
+    return (_ctxs[id].prefix_len > 0);
```

Comment:
(since `ltime == 0` is a legal value to decompress a context according to https://tools.ietf.org/html/rfc6775#section-5.4.3)